library/src/loader_impl.cxx

```diff
@@ -224,8 +224,13 @@ loader& loader_impl::loadGeometry(const std::string& filePath, bool reset)
   }
 
   f3d::reader* reader = f3d::factory::instance()->getReader(filePath);
-  if (!reader)
+  if (reader)
+  {
+    log::debug("The best reader for \"" + filePath + "\" is \"" + reader->getName() + "\"");
```

Comment:
thinking back on it, maybe it should be something like:

"found a reader for filepath: reader->getName()"

wdyt ?

Reply:
Yeah, no need to display reader name then?